db/schema.rb

```diff
@@ -148,7 +148,7 @@
     t.string "magic_link_token"
     t.datetime "magic_link_token_sent_at"
     t.boolean "hide_in_reporting", default: false, null: false
-    t.bigint "course_from_find_id"
+    t.integer "course_from_find_id"
```

Comment:
Looks unintentional this? I think this has happened before to someone.

Reply:
That might be running different versions of Postgres - need to check that...

Reply:
I get the `integer` in `schema.rb` when I run this migration https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/db/migrate/20200120120940_add_course_from_find_id_to_candidate_model.rb

I don't know why it's different for me, maybe because I'm still running Postgres 10 though it makes no sense - `bigint` is supported in older versions. (I think we are still on 9 in production).

Anyway I've repaved my database and then rerun my migration to get around it for now.